integration_tests/mockApis/auth.ts

```diff
@@ -9,7 +9,7 @@ const createToken = () => {
     user_name: 'USER1',
     scope: ['read'],
     auth_source: 'nomis',
-    authorities: [],
+    authorities: ['ROLE_ACTIVITY_HUB'],
     jti: '83b50a10-cca6-41db-985f-e87efb303ddb',
```

Comment:
Not sure this is right?  Any valid prison user can enter the service. Only certain routes/tiles require ROLE_ACTIVITY_HUB.